drivers/hmis/spec/requests/hmis/assessments/submit_household_assessments_spec.rb

```diff
@@ -59,7 +59,7 @@
     {
       form_definition_id: fd1.id,
       values: { 'linkid-date' => 2.weeks.ago.strftime('%Y-%m-%d') },
-      hud_values: { 'informationDate' => 2.weeks.ago.strftime('%Y-%m-%d') },
+      hud_values: { 'assessmentDate' => 2.weeks.ago.strftime('%Y-%m-%d') },
```

Comment:
there is no informationDate on custom assessment so this param silently discarded. We are no longer skipping custom assessment fields which meant this test failed with an exception. Use AssessmentDate instead